analysis/tests/src/expected/BrokenParserCases.res.txt

```diff
@@ -0,0 +1,62 @@
+Dump AST src/BrokenParserCases.res 1:22
+
+Source:
```

Comment:
If the source is repeated, how about also indicating what line number this is?

Reply:
Yes, for sure! I need to re-do this part though, it's very hacky and breaks for a lot of cases. I did however find it quite convenient to be able to just scan the resulting txt and have everything I need to think about the AST there.